src/Resource/CloudAccount/Endpoint.php

```diff
@@ -148,6 +151,139 @@ protected function _waitUntilPhpVersion(
     };
   }
 
+  /**
+   * Create a backup
+   *
+   * @param Entity An instance of cloud account entity.
+   * @return Backup
+   * @throws ApiException If request fails
+   */
+  public function createBackup(Entity $entity) : Backup {
+    $this->wait(null);
+    $response = $this->_client->request(
+      'POST',
+      self::_URI . "/{$entity->getId()}/backup"
+    );
+
+    return $this->getModel(Backup::class)->sync($response);
+  }
+
+  /**
+   * Return a list of backups
+   *
+   * @return Backup
+   * @throws ApiException If request fails
+   */
+  public function getBackups(Entity $entity) : Collection {
+    $this->wait(null);
+    $collection = new Collection(Backup::class);
+    
+    foreach ($this->_fetchBackupList($entity) as $backup) {
+      $collection->add($this->getModel(Backup::class)->sync($backup));
+    }
+
+    return $collection;
+  }
+
+  /**
+   * Return a specific backup
+   *
+   * @param string $file_name The unique file name for the backup to retrieve.
+   * @return Backup
+   * @throws ApiException If request fails
+   */
+  public function getBackup(Entity $entity, string $file_name) : Backup {
+    $this->wait(null);
+    return $this->_findBackup($entity, $file_name);
+  }
+
+  /**
+   * Download a specific backup
+   *
+   * @param string $file_name The unique file name for the backup to retrieve.
+   * @param string $path the directory to store the download in.
+   *
+   * @return Promise
+   * @throws ApiException If request fails
+   * @throws Exception
+   */
+  public function downloadBackup(Entity $entity, string $file_name, string $path) {
+    $this->wait(null);
+
+    if (! file_exists($path) || ! is_dir($path)) {
+      throw new Exception('##LG_INVALID_PATH##');
+    }
+
+    $path = trim($path);
+    if (substr($path, -1) !== DIRECTORY_SEPARATOR) {
+      $path .= DIRECTORY_SEPARATOR;
+    }
+
+    $save_to = $path . $file_name;
+
+    if (file_exists($save_to)) {
+      throw new Exception('##LG_FILE_ALREADY_EXISTS##');
+    }
+
+    $stream = fopen($save_to,'w');
+
+    $this->_client->request(
+      'GET',
+      $this->_findBackup($entity, $file_name)->get('download_url'),
+      [
+        'cookies' => (new CookieJar()),
```

Comment:
doesn't need to be ( wrapped in parens ) here

do you think this is worth doing anyway (more clear)?

Reply:
it is more clear to me, yes.